cmd/stomp-tool/main.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright (c) 2018 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
```

Comment:
When using libraries that use semantic versioning please update the `Gopkg.toml` file to explicitly specify the version, and update the `Gopkg.lock` file accordingly:

```
[[constraint]]
  name = "github.com/spf13/pflag"
  version = "1.0.1"

[[constraint]]
  name = "github.com/spf13/cobra"
  version = "0.0.2"
```

Reply:
Done :+1: